lib/components/modals/modalview.coffee

```diff
@@ -125,6 +127,8 @@ module.exports = class KDModalView extends KDView
           overlay  : yes
           content  : "<div class='modalformline'><p>#{helpContent}</p></div>"
 
+    super
```

Comment:
Shouldn't you also pass `e` argument to `super`? For the sake of consistency, at least.